sherpa/models/tests/test_caching.py

```diff
@@ -175,6 +175,7 @@ def test_evaluate_cache1dint():
     x1hi = np.array([-10, 1, -8])
 
     mdl = Polynom2D()
+    mdl.cache = 12
```

Comment:
The test name here is wrong, shouldn't it be `test_evaluate_cache2d`? I haven't checked to see if there is a repeated `test_evaluate_cache1dint` that this shadows, or whether we need one.

Also, should this tests sending in an integrated dataset but with the `integrate` flag set to `False` - which is an important test. Do we also need to test with it set to `True`, or does another test already cover this?